bin/minifollowups/pycbc_page_snglinfo

```diff
@@ -53,6 +53,11 @@ parser.add_argument('--ranking-statistic', default='newsnr',
          "'newsnr'")
 
 args = parser.parse_args()
+
+if args.trigger_id is not None and args.n_loudest is not None:
```

Comment:
how about using mutually_exclusive_group or whatever it's called?

Reply:
Done